csrc/scheduler/registry.cpp

```diff
@@ -1949,6 +1932,8 @@ class PersistentKernelScheduler : public SchedulerEntry {
     auto properties = scheduler_utils::getReductionProperties(
         fusion, runtime_info, reference_tv);
 
+    const int64_t warp_size = at::cuda::warp_size();
```

Comment:
Is this the same value as `dev_prop->warpSize`?

Reply:
yes. changed to `at::cuda::getCurrentDeviceProperties()->warpSize`